features/README.md

```diff
@@ -1 +1,22 @@
-TODO
\ No newline at end of file
+# `enviroment` setup
+
+This scipt sets up an enviorment (docker container) for testing and after testing has been completed removes the container and temporary directories.
+
+## `before_all` function
```

Comment:
We don't need to document all of these functions, as the comments in `environment.py` explain what each function does, so you can delete them.